pkg/cmd/attestation/verify/verify.go

```diff
@@ -25,18 +25,22 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 		Args:  cmdutil.ExactArgs(1, "must specify file path or container image URI, as well as one of --owner or --repo"),
 		Short: "Verify an artifact's integrity using attestations",
 		Long: heredoc.Docf(`
-			### NOTE: This feature is currently in beta, and subject to change.
-
 			Verify the integrity and provenance of an artifact using its associated
 			cryptographically signed attestations.
 
-			The command requires either:
+			In order to verify an attestation, you must validate the identity of the Actions
+			workflow that produced the attestation (a.k.a. the signer workflow). Given this
+			identity, the verification process checks the signatures in the attestations,
+			and confirms that the attestation refers to provided artifact.
```

Comment:
I wonder if we can structure this a bit more to help people develop their mental model of what's happening. Something like:

```
In order to verify an attestation, you need:
 * the artifact the attestation references
 * the attestation document (also called a bundle)
 * a policy describing where you're expecting the artifact came from (e.g. a specific GitHub owner or repository)

To specify the artifact, the command requires: ...

To specify the attestation document, the commend requires `--owner`, `--repo`, or `--bundle`. If you're using reusable workflows, the owner and repo will refer to the calling workflow.

There are several ways to specify the policy. If your signing doesn't take place in a reusable workflow, `--owner` and `--repo` will also indicate where you're expecting the build took place. However, if you are signing in a reusable workflow that's in a different repository than your calling workflow, use `--signer-repo`. You can also specify the exact signing workflow with `--signer-workflow`.
```

There's a lot of nuance we're trying to cram in here!

Reply:
i feel like at this level of detail we might be good to link to a guide/help guide, and policy is kinda too broad.

the main goal for this PR was to establish clarity around what to do with reusable workflows; thru that lens & given our current timeline i think this is a stepwise improvement & we can revise as we go along.

Reply:
Kiss my ass